src/roman-numerals/index.js

```diff
@@ -4,6 +4,33 @@
  * @param {string} roman The all-caps Roman numeral between 1 and 3999 (inclusive).
  * @returns {number} The decimal equivalent.
  */
-function romanToDecimal(roman) {}
+function romanToDecimal(roman) {
+    if(roman == null) return -1;
+    var number = char_to_int(roman.charAt(0));
```

Comment:
Prefer camelCasing when writing js variables. 

Rather than `char_to_int`, instead `charToInt`, its a convention though, not a rule.